website_anchor_smooth_scroll/static/src/js/website_anchor_smooth_scroll.js

```diff
@@ -19,7 +19,10 @@ function website_anchor_smooth_scroll(event) {
     });
 }
 
-(function ($) {
+require('web.dom_ready');
+
+function () {
     // Apply to all links that start with `#`
-    $("a[href^='#']").live("click", website_anchor_smooth_scroll);
-})(jQuery);
+    $("a[href^='#']:not([href=#])").on("click", website_anchor_smooth_scroll);
```

Comment:
This function is being declared but never executed, just execute this line, without wrapping it in a function please 😊